src/node/history.h

```diff
@@ -353,7 +372,22 @@ namespace ccf
         primary_sig,
         endorsed_cert);
 
+      constexpr int64_t vds_merkle_tree = 2;
+      const auto pheaders = {
+        // 1. VDS
+        ccf::crypto::cose_params_int_int(
+          ccf::crypto::COSE_PHEADER_KEY_VDS, vds_merkle_tree),
+        // 2. TxID
+        ccf::crypto::cose_params_string_string(
+          ccf::crypto::COSE_PHEADER_KEY_TXID, txid.str())
+        // 3. Key digest
+        // TO
```

Comment:
@achamayou let's discuss missing bits.

- what do we put into the protected header for key check
- anything else we want to provide via unprotected headers?